From 762afc4c587e45da585a0fa1890430f8b13d4509 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 22 Jul 2026 23:58:08 +0200
Subject: [PATCH] configure.ac: Set JSONC_LIBS and JSONC_CFLAGS

---
 configure.ac | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure.ac b/configure.ac
index a52c4725..9fcbbc8d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,6 +12,10 @@ AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([foreign
                   subdir-objects])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])]) #Backward compatible setting of "silent-rules"
+m4_divert_text([INIT_PREPARE], [
+export JSONC_LIBS="-L/app/extensions/o-charts_pi/lib -ljson-c"
+export JSONC_CFLAGS="-I/app/extensions/o-charts_pi/include"
+])
 
 AC_CONFIG_HEADERS([config.h])
 
-- 
2.54.0

